src/fundus/publishers/my/__init__.py

```diff
@@ -0,0 +1,15 @@
+from fundus.publishers.base_objects import PublisherEnum, PublisherSpec
+from fundus.publishers.my.malay_mail import MalayMailParser
+from fundus.scraping.url import RSSFeed, Sitemap
+
+
+class MY(PublisherEnum):
+    MalayMail = PublisherSpec(
+        name="Malay Mail",
+        domain="https://www.malaymail.com/",
+        sources=[
+            Sitemap("https://www.malaymail.com/sitemap.xml"),
+            RSSFeed("https://www.malaymail.com/feed/rss/malaysia"),
```

Comment:
```suggestion
            RSSFeed("https://www.malaymail.com/feed/rss/"),
```
This seems to be a broader RSSFeed. While it still seems to have the same articles as the /malaysia/ feed, it also has a couple extra